src/components/Homepage/HomepageStatistics.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+	<section class="statistics section">
+		<div class="row">
+			<h1
+				v-html="statsHeadline"
+				class="small-12 large-6 columns statistics__header huge green-emphasis"
+			>
+			</h1>
+			<span class="statistics__video-wrapper small-12 large-6 columns">
+				<video
+					:src="statsVideo.file.url"
+					class="statistics__video-wrapper--video"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+			</span>
+		</div>
+		<div class="row statistics__stat-block">
+			<div
+				v-for="(statBlock, index) in statsBlockText"
+				:key="statBlock.key"
+				class="small-12 medium-4 columns"
+			>
+				<kv-contentful-img
+					v-if="statBlock.image.url"
+					:class="'statistics__stat-block--icon statistics__stat-block--icon-' + index"
+					:contentful-src="statBlock.image.url"
+					:alt="statBlock.image.description"
+					fallback-format="png"
+					:width="50"
```

Comment:
@ryan-ludwig How does the width I'm passing through here translate to the size of the img since I'm setting the width of each icon individually? We have a v-if="(width || height)" check within kv-contentful-img, so I've included this width in order to get the icons to display. 50 is about the average width of the icons I'm displaying.

Reply:
You're right that you'll always need either a width or a height set. It looks like all of those icons in figma are 48 x 48 if you're exporting it from that layer.
![image](https://user-images.githubusercontent.com/187937/110052004-ac137280-7d0b-11eb-9195-8723bc1341ff.png)

So I'd double check that the PNGs you are uploading to contentful are 96x96 (48x48 * 2), and then set width=48 and height=48